### PR TITLE
remove ArtifactWrappers from docs, test, standalone bucket

### PR DESCRIPTION
**Pull request by kmdeck**
## Purpose 
Removes artifact wrappers dependence from our docs, test, and  src except src/Artifacts.jl

## To-do
Future PR (ClimaArtifacts) - add these files to ClimaArtifacts
Future PR (ClimaLand) - update the functions in src/Artifacts.jl
Review

## Content
adds a few new files to srcs/Artifacts.jl
removes AW from docs, src/standalone/Bucket
In test/ we had "era5_t2m_sp_u10n_static_dataset_path" and "era5_t2m_sp_u10n_dataset_path" artifacts, but these were not being used anywhere, so I removed them. @juliasloan25 can confirm

Review checklist

I have:
- followed the codebase contribution guide: https://clima.github.io/ClimateMachine.jl/latest/Contributing/
- followed the style guide: https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/
- followed the documentation policy: https://github.com/CliMA/policies/wiki/Documentation-Policy
- checked that this PR does not duplicate an open PR.

In the Content, I have included 
- relevant unit tests, and integration tests, 
- appropriate docstrings on all functions, structs, and modules, and included relevant documentation.

----
- [X] I have read and checked the items on the review checklist.

### CODE REVIEW

File: src/Artifacts.jl [hunk 1]
```diff
@@ -221,6 +221,132 @@ function topmodel_data_path(; context = nothing)
...
+Data was originally collected by Lehmann, Peter, Shmuel Assouline, 
+and Dani Or. "Characteristic lengths affecting evaporative drying of 
+porous media." Physical Review E 77.5 (2008): 056309 and presented
+in Figure 8 of that work.
+
+https://doi.org/10.1103/PhysRevE.77.056309
+"""
+function lehmann_assouline_or2008_evaporation_data(; context = nothing)
+    evap_dataset = ArtifactWrapper(
+        @__DIR__,
```
Comment (Sbozzolo) [inline src/Artifacts.jl:242]: Do we mabye need `joinpath(@__DIR__, "..")?

Comment (kmdeck) [reply src/Artifacts.jl:242]: yes, I forgot! good catch

Comment (Sbozzolo) [reply src/Artifacts.jl:242]: Good to go once you fix that!

**Review by Sbozzolo — APPROVED** [review]

**Comment by juliasloan25** [comment]
Agreed that "era5_t2m_sp_u10n_static_dataset_path" and "era5_t2m_sp_u10n_dataset_path" weren't being used! Thank you Kat :)

**Comment by juliasloan25** [comment]
Is it correct that this is a step towards https://github.com/CliMA/ClimaLand.jl/issues/580, but doesn't close it because ArtifactWrappers is still being used in src/Artifacts.jl?

**Comment by Sbozzolo** [comment]
> Is it correct that this is a step towards #580, but doesn't close it because ArtifactWrappers is still being used in src/Artifacts.jl?

Yes, correct.

**Pull request merged**